components/mysql/actions/execute-stored-procedure/execute-stored-procedure.mjs

```diff
@@ -0,0 +1,41 @@
+import mysql from "../../mysql.app.mjs";
+
+export default {
+  key: "mysql-execute-stored-procedure",
+  name: "Execute Stored Procedure",
+  description: "Execute Stored Procedure. [See the docs here](https://dev.mysql.com/doc/refman/8.0/en/stored-programs-defining.html)",
+  type: "action",
+  version: "0.0.1",
+  methods: {
+    async executeStoredProcedure({
+      storedProcedure, params = [],
+    }) {
+      const sql = `CALL ${storedProcedure}(${params.join(",")});`;
```

Comment:
I think we should sanitize this guy here as well.
I would like to suggest moving it to `run` since this is called just here and `run` function is pretty small